README.md

```diff
@@ -1 +1,56 @@
-# Getting started with Cisco IOS-XE Smart licensing APIs
\ No newline at end of file
+# Getting Started with Cisco IOS-XE Smart licensing APIs
+
+Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
+
+For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+
+## Use Case Description
+
+For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+
+Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
+
+## Installation
+
+Python must be installed. Further the following Python packages are required:
```

Comment:
This script has been written using Python 3.8.5. 
Along with that python version, the following python modules are required:

Reply:
Noted.

Reply:
Noted; added this info